.moai/specs/SPEC-UPDATE-YAML-PRESERVE-001/acceptance.md

```
@@ -202,69 +223,119 @@ matches=$(grep -rn 'expected user_added to be dropped' --include='*_test.go' . |
+### AC-UYP-023 — `quality.yaml.tmpl` parses via node decoder (plan-audit D4)
+
+**Given** the `quality.yaml.tmpl` template, whose unquoted placeholders `{{.EnforceQuality}}` / `{{.TestCoverageTarget}}` cause the **map** decoder (`map[string]any`) to fail with `yaml: invalid map key`
+**When** `MergeYAML3Way` runs against the rendered-or-raw `quality.yaml` body using the **node** decoder
+**Then** the merge returns a nil error (the placeholder lands as scalar-nested-flow-mapping text rather than failing), AND the file's transition from always-2-way (pre-fix, silent `restore.go:139` fallback) to real-3-way (post-fix) is observable.
+
+This AC closes the D4 finding: the original SPEC never named `quality.yaml.tmpl`'s unparseability, and its fix-side effect (the file gets promoted from 2-way to 3-way) was entirely untested. Combined with the D5 deferral (whose wrong-base problem newly manifests for this file post-fix — see plan.md Decision D5 safety-argument point 4), this AC is what makes the `quality.yaml` path auditable rather than silent.
+
+```bash
+go test ./internal/cli/update/backup/ -run TestMergeYAML3Way_QualityTemplateParses -count=1 -v 2>&1 \
+  | tee /dev/stderr | grep -q '^--- PASS: TestMergeYAML3Way_QualityTemplateParses'
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)